wf-core-tck-runner/rest-tck/src/test/resources/tck-config.yml

```diff
@@ -0,0 +1,20 @@
+wildfly-configuration:
+  subsystem:
+    logging:
+      # Remove the level for the console handler to show all log messages in the console
+      console-handler:
+        CONSOLE:
+          level: !undefine
+      # Add TRACE logging for the org.jboss.resteasy logger as some errors get swallowed in the client and this makes
+      # it easier to see why things have failed.
+      logger:
+        org.jboss.resteasy:
+          level: TRACE
+    undertow:
+      server:
+        default-server:
+          http-listener:
+            default:
+              # We need to all HTTP methods to be allowed. By default, TRACE is not allowed
```

Comment:
@jamezp Is this alternate wording correct?

"# A TCK test needs all HTTP methods to be allowed.  By default, TRACE is not allowed."

Sorry to be nitpicky but I'm aiming for there to be no ambiguity in these yaml files as to whether a change is needed to allow the TCK to do things that the spec doesn't require an impl to provide OOTB, vs changes made for other reasons.

Reply:
That makes sense to me. I'll fix it now.